notation.go

```diff
@@ -169,6 +169,7 @@ func Verify(ctx context.Context, verifier Verifier, repo registry.Repository, op
 	}
 	errExceededMaxVerificationLimit := ErrorVerificationFailed{Msg: fmt.Sprintf("total number of signatures associated with an artifact should be less than: %d", opts.MaxSignatureAttempts)}
 	numOfSignatureProcessed := 0
+	userSignaureMediaType := opts.SignatureMediaType
```

Comment:
`opts.SignatureMediaType` should be an array to allow user to trust multiple signature formats. Also, an empty array here means they trust all.

Reply:
Added unit test. As for changing it to array, IMO it's not very necessary at this point since we only support two formats. An empty user input means supporting both of them. Besides, changing VerifyOption would require Ratify to change accordingly.

Reply:
If this is external facing method used by ratify, I would suggest changing this to array to avoid any breaking changes post RC1.

Reply:
I am confused, either we need to take this as an array or we should completely drop this functionality and always use `mediaType` present signature artifact manifest.